src/uikit.cs

```diff
@@ -21156,4 +21200,167 @@ interface UIKey : NSCopying, NSCoding {
+	}
+
+	[NoWatch, NoTV, iOS (13,4)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UIPointerRegionRequest {
+		
+		[Export ("location")]
+		CGPoint Location { get; }
+
+		[Export ("modifiers")]
+		UIKeyModifierFlags Modifiers { get; }
+	}
+
+	interface IUIPointerInteractionAnimating { }
+	
+	[NoWatch, NoTV, iOS (13,4)]
+	[Protocol]
+	interface UIPointerInteractionAnimating {
+
+		[Abstract]
+		[Export ("addAnimations:")]
+		void AddAnimations (Action animations);
+
+		[Abstract]
+		[Export ("addCompletion:")]
+		void AddCompletion (Action<bool> completion);
+	}
+
+	[NoWatch, NoTV, iOS (13,4)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UIPointerRegion : NSCopying {
+
+		[Export ("rect")]
+		CGRect Rect { get; }
+
+		[NullAllowed, Export ("identifier")]
+		NSObject Identifier { get; }
+
+		[Static]
+		[Export ("regionWithRect:identifier:")]
+		UIPointerRegion Create (CGRect rect, [NullAllowed] NSObject identifier);
+	}
+
+	[NoWatch, NoTV, iOS (13,4)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UIPointerStyle : NSCopying {
+
+		[Static]
+		[Export ("styleWithEffect:shape:")]
+		UIPointerStyle Create (UIPointerEffect effect, [NullAllowed] UIPointerShape shape);
+
+		[Static]
+		[Export ("styleWithShape:constrainedAxes:")]
+		UIPointerStyle Create (UIPointerShape shape, UIAxis axes);
+
+		[Static]
+		[Export ("hiddenPointerStyle")]
+		UIPointerStyle CreateHiddenPointerStyle ();
+	}
+
+	[NoWatch, NoTV, iOS (13,4)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UIPointerEffect : NSCopying {
+
+		[Export ("preview", ArgumentSemantic.Copy)]
+		UITargetedPreview Preview { get; }
+
+		[Static]
+		[Export ("effectWithPreview:")]
+		UIPointerEffect Create (UITargetedPreview preview);
+	}
+
+	[NoWatch, NoTV, iOS (13,4)]
+	[BaseType (typeof (UIPointerEffect))]
+	interface UIPointerHighlightEffect {
+
+	}
+
+	[NoWatch, NoTV, iOS (13,4)]
+	[BaseType (typeof (UIPointerEffect))]
+	interface UIPointerLiftEffect {
+
+	}
+
+	[NoWatch, NoTV, iOS (13,4)]
+	[BaseType (typeof (UIPointerEffect))]
+	interface UIPointerHoverEffect {
+
+		[Export ("preferredTintMode", ArgumentSemantic.Assign)]
+		UIPointerEffectTintMode PreferredTintMode { get; set; }
+
+		[Export ("prefersShadow")]
+		bool PrefersShadow { get; set; }
+
+		[Export ("prefersScaledContent")]
+		bool PrefersScaledContent { get; set; }
+	}
+
+	[NoWatch, NoTV, iOS (13,4)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UIPointerShape : NSCopying {
+
+		[Static]
+		[Export ("shapeWithPath:")]
+		UIPointerShape Create (UIBezierPath path);
+
+		[Static]
+		[Export ("shapeWithRoundedRect:")]
+		UIPointerShape Create (CGRect rect);
+
+		[Static]
+		[Export ("shapeWithRoundedRect:cornerRadius:")]
+		UIPointerShape Create (CGRect rect, nfloat cornerRadius);
+
+		[Static]
+		[Export ("beamWithPreferredLength:axis:")]
+		UIPointerShape CreateBeam (nfloat length, UIAxis axis);
```

Comment:
`preferredLength` instead of `length` since it suggest it might not be the actual length used

Reply:
done